src/mercury_engine_data_structures/formats/brfld.py

```diff
@@ -17,16 +17,16 @@ class Brfld(BaseResource):
     def construct_class(cls, target_game: Game) -> construct.Construct:
         return standard_format.game_model("CScenario", "49.0.2")
 
-    def actors_for_layer(self, name: str) -> dict:
-        return self.raw.Root.pScenario.rEntitiesLayer.dctSublayers[name].dctActors
+    def actors_for_sublayer(self, sublayer_name: str, layer_name: str = "rEntitiesLayer") -> dict:
+        return self.raw.Root.pScenario[layer_name].dctSublayers[sublayer_name].dctActors
 
-    def all_layers(self) -> Iterator[str]:
-        yield from self.raw.Root.pScenario.rEntitiesLayer.dctSublayers.keys()
+    def sublayers_for_layer(self, layer_name: str) -> Iterator[str]:
```

Comment:
missing default

Reply:
I intentionally left out the default here, but I'll add one because upon further thought, it's definitely better to have one.